test/demo/raiseIntent/withTarget.inttest.ts

```diff
@@ -131,8 +132,8 @@ listener to be added', async () => {
                         );
 
                         await expect(raiseIntentPromise).toThrowFDC3Error(
-                            ResolveError.IntentTimeout,
-                            `Timeout waiting for intent listener to be added for intent: ${validIntent.type}`
+                            RaiseIntentError.SendIntentNoHandler,
+                            `No intent handler added for intent: ${validIntent.type}`
```

Comment:
Do the usual "maturity" rules apply here?

Will reject instantly with "no handler" error if a mature app has no listeners, if an immature app has no listeners, it's given the remainder of it's "startup" period in which to add one?

Reply:
Maturity rules still apply, I just decided not to phrase this as a timeout, given that we're using that term for timeouts on client calls, and it doesn't quite fit so well when we're making a call to a long-running app rather than one that's just started